citesphere-importer/src/main/java/edu/asu/diging/citesphere/importer/core/service/parse/iterators/CrossRefIterator.java

```diff
@@ -0,0 +1,254 @@
+        typeMap.put("dataset", Publication.DATASET);
+        typeMap.put("book-series", Publication.BOOK_SERIES);
+    }
+
+    private ContainerMeta parseJournalMeta(Item item) {
+        ContainerMeta meta = new ContainerMeta();
+        meta.setContainerTitle(item.getContainerTitle().get(0));
+        meta.setPublisherName(item.getPublisher());
+        meta.setPublisherLocation(item.getPublisherLocation());
+        List<Issn> issnList = new ArrayList<Issn>();
+        if(item.getIssnType() != null) {
+            for(IssnType issnType : item.getIssnType()) {
+                Issn issn = new Issn();
+                issn.setIssn(issnType.getValue());
+                issn.setPubType(issnType.getType());
+                issnList.add(issn);
+            }
+        }
+        meta.setIssns(issnList);
+        return meta;
+    }
+
+    private ArticleMeta parseArticleMeta(Item item) {
+        ArticleMeta meta = new ArticleMeta();
+        meta.setArticleTitle(item.getTitle().get(0));
+        meta.setShortTitle(String.join(", ", item.getTitle().subList(1, item.getTitle().size())));
+        List<Contributor> contributors = new ArrayList<>();
+        // List of authors
+        if(item.getAuthor() != null) {
+            contributors.addAll(mapPersonToContributor(item.getAuthor(), ContributionType.AUTHOR));
+        }
+        // List of editors
+        if(item.getEditor() != null) {
+            contributors.addAll(mapPersonToContributor(item.getEditor(), ContributionType.EDITOR));
+        }
+        // List of translators
+        if(item.getTranslator() != null) {
+            contributors.addAll(mapPersonToContributor(item.getTranslator(), ContributionType.TRANSLATOR));
+        }
+        // List of chair
+        if(item.getChair() != null) {
+            contributors.addAll(mapPersonToContributor(Arrays.asList(item.getChair()), ContributionType.CHAIR));
+        }
+        meta.setContributors(contributors);
+
+        meta.setAuthorNotesCorrespondence(null);
+        ArticlePublicationDate publicationDate = new ArticlePublicationDate();
+        List<Integer> dateParts = item.getPublished().getIndexedDateParts();
+        if(dateParts != null) {
+            publicationDate.setPublicationDate(dateParts.get(2).toString());
+            publicationDate.setPublicationMonth(dateParts.get(1).toString());
+            publicationDate.setPublicationYear(dateParts.get(0).toString());
+        }
+        meta.setPublicationDate(publicationDate);
+        meta.setVolume(item.getVolume());
+        meta.setIssue(item.getIssue());
+        meta.setPartNumber(item.getPartNumber());
+        meta.setFirstPage(item.getPage());
+        meta.setSelfUri(item.getUrl());
+        meta.setArticleAbstract(item.getAbstractText());
+        meta.setLanguage(item.getLanguage());
+        ReviewInfo review = new ReviewInfo();
+        if (item.getReview() != null) {
+            review.setFullDescription(item.getReview().getCompetingInterestStatement());  
```

Comment:
is full description the same as competing interest statement?

Reply:
Yes.